january-slates/readme.md

```diff
@@ -13,11 +13,11 @@ This dataset is needed to power automatic detection of slate information in the
 Many of the collection pieces have incomplete or unverified information about what the video is. 
 This detected information could be used to verify and update metadata about that collection piece.
 
-### Project Information
+### Specs
 * Annotation project name - `january-slates`
 * Annotator Demographics
     * Number of annotators - Two
-    * Occupational description - Two Public Broadcasting Organization Media Librarians and Volunteers
+    * Occupational description - Two Public Broadcasting Organization git Media Librarians and Volunteers
```

Comment:
typo?

Reply:
Why didn't this get fixed before merge?

Reply:
This got confused as fixed already. Will have to fix on next commit.

Reply:
fixed in 34978bccf3cb9d6e3b0ccfe3c6c0cf15d2403f2d